h2/stream.py

```diff
@@ -1071,30 +1074,26 @@ def _build_hdr_validation_flags(self, events):
         Constructs a set of header validation flags for use when normalizing
         and validating header blocks.
         """
-        try:
-            is_trailer = isinstance(
-                events[0], (_TrailersSent, TrailersReceived)
-            )
-            is_response_header = isinstance(
-                events[0], (_ResponseSent, ResponseReceived)
+        is_trailer = isinstance(
+            events[0], (_TrailersSent, TrailersReceived)
+        )
+        is_response_header = isinstance(
+            events[0],
+            (
+                _ResponseSent,
+                ResponseReceived,
+                InformationalResponseReceived
```

Comment:
Seeing this makes me wonder: should `InformationalResponseReceived` be a subclass of `ResponseReceived`?